Documentation/project-docs/pull-request-validation.md

```diff
@@ -0,0 +1,16 @@
+In the .NET Platform repository we agreed on the following policies before a Pull Request can be merged:
+
+1. Every PR needs to be **approved** by a team member with write access to the repository.
+2. Unless changes aren't protected by **Continuous Integration** (CI), i.e. Markdown edits, CI must have run recently.
```

Comment:
i.e. => e.g.

Reply:
Also, can you define "recently"?

Reply:
Agree, maybe phrase like "within X days unless no commits have been merged.  In cases where other PRs have been merged into the same library since CI was run, you may wish to rerun in order to get fresh results and avoid build breaks."

Reply:
thanks I added the phrase.